lib/rails/rollbar_runner.rb

```diff
@@ -49,7 +49,7 @@ def eval_runner
     def legacy_runner
       string_to_eval = File.read(runner_path)
 
-      ::Rails.module_eval(<<-EOL, __FILE__, __LINE__ + 2)
+      ::Rails.module_eval(<<-EOL, __FILE__, __LINE__ + 1)
```

Comment:
Not a blocker but there's enough indirection here that I don't fully know why we're adding 1 to `__LINE__`. I assume it is set to 1 to match the location of `#{string_to_eval}`? If that's the case, was the 2 a bug?

This might benefit from a small comment.

Reply:
Great question. It sets location information in the event of an error with backtrace. The `2` seems like it was a bug, though a minor one. 

Yes this looks pretty esoteric, and I think what looks unexpected isn't the file/line args for `module_eval`. After all, those can be looked up quickly if you don't know how `module_eval` works. IMO it's the opening a heredoc in the first arg and finishing it on the following lines! Maybe a comment is deserved here.